pkg/controller/cronjob/metrics/metrics.go

```diff
@@ -31,7 +31,7 @@ var (
 			Subsystem:      CronJobControllerSubsystem,
 			Name:           "cronjob_job_creation_skew_duration_seconds",
```

Comment:
agreed, since this is a metric from the cronjob controller, it is explicit that the jobs it gives information about are from a cronjob so adding this additional prefix is redundant.